optics-compat/optics-compat.cabal

```diff
@@ -0,0 +1,23 @@
+name:          optics-compat
+version:       0.1
+license:       BSD3
+license-file:  LICENSE
+build-type:    Simple
+maintainer:    optics@well-typed.com
+author:        Andrzej Rybczak
+cabal-version: >=1.10
+tested-with:   ghc ==8.0.2 || ==8.2.2 || ==8.4.4 || ==8.6.3
+synopsis:      Utils for compatibility with van Laarhoven optics
+category:      Data, Optics, Lenses
+description:
+  This package provides utils for compatibility with van Laarhoven optics
+
+library
+  default-language: Haskell2010
+  hs-source-dirs:   src
+  ghc-options:      -Wall
+  build-depends: base          >=4.9     && <5
+               , optics-core   >=0.1     && <1
+               , profunctors   >=5.0     && <6.0
```

Comment:
Presumably we'll leave `optics-compat` out of `optics` so as not to impose a dependency on `profunctors`?

Reply:
Yes.